docs/knowledgebase/advanced/ss58-address-format.md

```diff
@@ -2,92 +2,95 @@
 title: SS58 Address Format
 ---
 
-SS58 is a simple address format designed for Substrate based chains. There's no problem with using
-other address formats for a chain, but this serves as a robust default. It is heavily based on
-Bitcoin's Base-58-check format with a few alterations.
+SS58 is a simple address format designed for Substrate based chains, primarily 
+inspired by the [Bitcoin Base58Check](https://en.bitcoin.it/wiki/Base58Check_encoding) format with a few alterations.
+Substrate developers are free to use the address format of their choice, but this serves as a robust default. 
+The purpose of this article is to outline existing solutions to validating a SS58 address.
```

Comment:
It's not the purpose of this article. It's the purpose of this PR.

Reply:
My bad, this was I mistake. I'll just suggest my changes in review mode.